src/brevitas/graph/calibrate.py

```diff
@@ -24,7 +24,11 @@
 from .base import Transform
 
 __all__ = [
-    'ClipFloatWeights', 'DisableEnableQuantization', 'bias_correction_mode', 'calibration_mode']
+    'ClipFloatWeights',
+    'DisableEnableQuantization',
+    'bias_correction_mode',
+    'calibration_mode',
+    'allow_unexpected_bias_keys']
```

Comment:
Let's rename `allow_unexpected_bias_keys` to `load_quant_model`.
In the future we can use this as a more general entrypoint to load checkpoint with special requirements.

For example, at the moment we have a global flag to ignore missing keys when loading a floating point dict in a quantized model, and it's up to the user to enable that flag. 

This is a matter for another PR anyway.